sdk/storage/azure-storage-blobs/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 - `UserDelegationKey` was changed to a member of `GetUserDelegationKeyResult` rather than a typedef like before.
 - `AccessType` in `CreateBlobContainerOptions` was changed to non-nullable.
 - `ListType` in `GetBlockListOptions` was changed to non-nullable.
+- Added `BlobLeaseClient`, all lease related APIs are moved to `BlobLeaseClient`.
```

Comment:
nit: I recommend using descriptions like these as the **PR titles** rather than short sentence fragments.